### PR TITLE
Add Spectrum badge

### PR DESCRIPTION
**Pull request by mxstbr**
🎉 https://spectrum.chat/styleguidist

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -5,8 +5,7 @@
 **Isolated React component development environment with a living style guide**
 
 [![Build Status](https://travis-ci.org/styleguidist/react-styleguidist.svg)](https://travis-ci.org/styleguidist/react-styleguidist)
-[![npm](https://img.shields.io/npm/v/react-styleguidist.svg)](https://www.npmjs.com/package/react-styleguidist)
-[![Join the chat at https://gitter.im/styleguidist/styleguidist](https://badges.gitter.im/styleguidist/styleguidist.svg)](https://gitter.im/styleguidist/styleguidist?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/styleguidist)[![Join the chat at https://gitter.im/styleguidist/styleguidist](https://badges.gitter.im/styleguidist/styleguidist.svg)](https://gitter.im/styleguidist/styleguidist?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (sapegin) [inline Readme.md]: Please add a new line between the badges :-)

Comment (mxstbr) [reply Readme.md]: Oh damn I totally typo'ed right there.

Do you want me to remove the Gitter badge? It seems redundant if you're using Spectrum...

Comment (sapegin) [reply Readme.md]: Let’s keep it for now. Gitter seems better for quick questions and you can login with GitHub account.

Comment (mxstbr) [reply Readme.md]: Ohh we haven't enabled the watercooler for you!

We built a new feature recently that adds a irc-style "General chatroom" without any specific topic to your community which we're currently alpha-testing with a couple communities. Looks like this: https://spectrum.chat/thread/cfc324fa-13cc-4353-af00-0bbf0f611889

Want me to enable that for you?

Comment (brianlovin) [reply Readme.md]: The watercooler is live! Here you go: https://spectrum.chat/styleguidist?thread=6358bb3b-d2e0-4fef-88cb-7aa9d0c0933a

Comment (sapegin) [reply Readme.md]: And now I see “We weren’t able to load this community.” 😢

Comment (mxstbr) [reply Readme.md]: Yep sorry, that's my bad, I'm on it! Should be resolved shortly :blush:

**Pull request merged**